pkg/actions/label/label.go

```diff
@@ -22,13 +18,11 @@ type Manager struct {
 	clusterName string
 }
 
-func New(cfg *api.ClusterConfig, ctl *eks.ClusterProvider) *Manager {
-	sc := manager.NewStackCollection(ctl.Provider, cfg)
-	mc := managed.NewService(ctl.Provider, sc, cfg.Metadata.Name)
```

Comment:
gah but now the same identical setup lines need to happen in 3 places. sacrificed for the greater good i suppose 😭